lightning/src/ln/offers_tests.rs

```diff
@@ -1260,7 +1260,7 @@ fn fails_creating_or_paying_for_offer_without_connected_peers() {
 	let absolute_expiry = alice.node.duration_since_epoch() + MAX_SHORT_LIVED_RELATIVE_EXPIRY;
 	match alice.node.create_offer_builder(Some(absolute_expiry)) {
 		Ok(_) => panic!("Expected error"),
-		Err(e) => assert_eq!(e, Bolt12SemanticError::MissingPaths),
+		Err(e) => assert_eq!(e, Bolt12CreationError::InvalidSemantics(Bolt12SemanticError::MissingPaths)),
```

Comment:
Likewise, these changes should go in the commits that added `Bolt12CreationError`.